build_tools/bump_submodules.py

```diff
@@ -170,7 +235,32 @@ def run(args: argparse.Namespace, fetch_args: list[str], system_projects: list[s
             ["git", "push", "-u", "origin", args.branch_name],
             cwd=THEROCK_DIR,
         )
-
+        new_hashes = get_hashes_for_component(component)
+        old_h, new_h, changed_path = detect_hash_change(old_hashes, new_hashes)
+        if not old_h or not new_h:
+            log("WARNING: No submodule hash changes detected for the component.")
+            pr_title = f"Bump {component} from unknown to unknown"
+        else:
+            pr_title = f"Bump {component} from {old_h[:7]} to {new_h[:7]}"
+        pr_body = f"Bump happened on {datetime.today().strftime('%m%d%Y')}\n\n"
+        pr_body += "### Submodule changes:\n"
+        for sm in new_hashes:
+            pr_body += f"- `{sm}`: {old_hashes.get(sm)} → {new_hashes.get(sm)}\n"
```

Comment:
This title and body should go in the _commit_ as well, not just the PR. I think the default behavior for a PR is to take the commit title and body.